config/locales/en.yml

```diff
@@ -873,19 +873,36 @@ en:
     articles_edited: Articles Edited
     articles_edited_generic: Pages Edited
     articles_edited_wikidata: Items Edited
+    article_scoped_program_info:
+        Your course is an 'Article Scoped Program', in which only assigned articles and categories
```

Comment:
This is an 'Article Scoped Program', which only counts edits to assigned articles and categories.

Reply:
Done!